internal/mysql/example/sqlc.json

```diff
@@ -2,8 +2,8 @@
   "version": "1",
   "packages": [
     {
-      "name": "teachersDB",
-      "database": "mysql",
+      "name": "db",
```

Comment:
This example should be moved into the top-level `examples` directory.

Reply:
Do you object to removing the example altogether? I'm thinking it's best to have consistent examples across engines. So the `booktest` works great for this purpose.